src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import { NavLink } from 'react-router-dom';
+
+export function Button({
+  className = 'btn',
+  text = '',
+  href = '',
+  children,
+  onClick,
+}) {
+  return (
+    <>
+      {href !== '' ? (
```

Comment:
```suggestion
      {href  ? (
```
Then this would look like this ^

Reply:
👍